### PR TITLE
Add on_failure handler for Zeek ingest pipelines. Fix one field name …

### PR DESCRIPTION
**Pull request by alakahakai**
* Address issue 11004 - Add on_failure handler for Zeek ingest pipelines. 
* Fix one field name error for notice and add an additional test

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -161,6 +161,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Improve detection of file deletion on Windows. {pull}10747[10747]
 - Fix errors in filebeat Zeek dashboard and README files. Add notice.log support. {pull}10916[10916]
 - Fix a bug when converting NetFlow fields to snake_case. {pull}10950[10950]
+- Add on_failure handler for Zeek ingest pipelines. Fix one field name error for notice and add an additional test 
+  case. {{issue}11004[11004] pull}11105[11105]
```
Comment (webmat) [inline CHANGELOG.next.asciidoc]: Check out your asciidoc links here ;-)

File: x-pack/filebeat/modules.d/zeek.yml.disabled [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Module: zeek
+# Docs: https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-zeek.html
```
Comment (webmat) [inline x-pack/filebeat/modules.d/zeek.yml.disabled]: Should we use `/current/` instead of `/master/` here?

Comment (andrewkroh) [reply x-pack/filebeat/modules.d/zeek.yml.disabled]: I think these headers are generated based on the [documentation branch](https://github.com/elastic/beats/blob/master/libbeat/docs/version.asciidoc#L2) which is master so this is correct.

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (andrewkroh) [reply CHANGELOG.next.asciidoc]:
```suggestion
  case. {issue}11004[11004] {pull}11105[11105]
```

**Pull request merged**